package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stellar/typescript-wallet-sdk",
-  "version": "1.0.0-alpha.2",
+  "version": "1.1.0-alpha.3",
```

Comment:
NIT: for accurate [semver](https://semver.org/), i believe this should be `1.1.0-alpha.0`

Reply:
Should we name it beta now?

Reply:
I set it as `1.1.0-alpha.3` because our [current npm version](https://www.npmjs.com/package/@stellar/typescript-wallet-sdk?activeTab=versions) is `1.1.0-alpha.2`

But yeah, I think naming it beta sounds good!

Should we go with `1.0.0-beta.0` ?

Reply:
Ah I see - yes, let's do `1.0.0-beta.0`!